playbooks/roles/tower/tasks/bootstrap_credentials.yml

```diff
@@ -40,6 +40,7 @@
     ssh_key_data: '/tmp/tower_ssh'
     ssh_key_unlock: '{{ tower_ssh_password }}'
     username: '{{ tower_ssh_user }}'
+    password: '{{ tower_ssh_user_pass }}'
```

Comment:
Can we change this variable name to `tower_ssh_user_password` to be consistent? For example we also have `tower_ssh_password ` listed.

Reply:
Done.